client/post-editor/editor-sidebar/index.jsx

```diff
@@ -39,37 +45,50 @@ export default class EditorSidebar extends Component {
 			type,
 			setPostDate,
 			isPostPrivate,
+			nestedSidebar,
+			toggleNestedSidebar,
```

Comment:
Ask yourself whether these props should come with default values. Not a rhetorical question, just something that I'm wondering myself, as an external reviewer.